docs/en/observability/logs-stream.asciidoc

```diff
@@ -28,75 +28,62 @@ On your host, download and extract the installation package that corresponds wit
-. Replace `username: '${ES_USERNAME}'` and `password: '${ES_PASSWORD}'` with `api_key:` in the `elastic-agent.yml` file, and add the API key you created in the previous step. For example:
-+
 [source,yaml]
 ----
-[...]
 outputs:
   default:
     type: elasticsearch
-    hosts:
-      - 'https://da4e3a6298c14a6683e6064ebfve9ace.us-central1.gcp.cloud.es.io:443'
-    api_key: _Nj4oH0aWZVGqM7MGop8:349p_U1ERHyIc4Nm8_AYkw
-[...]
+    hosts: '<your-elasticsearch-url>:<port>'
+    api_key: 'your-api-key'
+inputs:
+  - id: your-log-id
+    type: filestream
+    streams:
+      - id: your-log-stream-id
+        data_stream:
+          dataset: generic
+        paths:
+          - /var/log/your-logs.log
 ----
-+
-NOTE: The format of the key is <id>:<key>. Make sure you selected *Beats* when you created your API key. Base64 encoded API keys are not currently supported in this configuration.
-
-[discrete]
-[[logs-stream-install-agent]]
-== Step 3: Install and start the {agent}
-With your configuration set, you're ready to install the {agent}. From the agent directory, run the the command that corresponds with your system to install the {agent} as a service. Do *not* enroll the agent in Fleet.
 
-NOTE: On macOS, Linux (tar package), and Windows, run the `install` command to
-install {agent} as a managed service and start the service. The DEB and RPM
-packages include a service unit for Linux systems with
-systemd, so just enable then start the service.
+To configure your {agent}, set the values for the fields in the example configuration:
 
-include::{ingest-docs-root}/docs/en/ingest-management/tab-widgets/run-standalone-widget.asciidoc[]
+. `hosts` – Your {es} URL and the port. On {ecloud}, this . 
+.. Copy the {es} endpoint and cluster ID and from your deployment's page.
++
+--
+[role="screenshot"]
+image::images/es-endpoint-cluster-id.png[{es} endpoint and cluster id location, 50%]
+--
++
+.. Your host URL is formatted as `https://CLUSTER_ID.REGION.CLOUD_PLATFORM.DOMAIN:PORT`. The default port is 443. For example, `https://ec47fc4d2c53414e1307e85726d4b9bb.us-east-1.aws.found.io:443`.
+. `api-key` – An API key to grant the agent access to {es}. To create an API key for your agent, see {fleet-guide}/grant-access-to-elasticsearch.html#create-api-key-standalone-agent[Create API keys for standalone agents].
++
+NOTE: The API key format should be <id>:<key>. Make sure you selected *Beats* when you created your API key. Base64 encoded API keys are not currently supported in this configuration.
+. `inputs.id` – A unique identifier for your input.
+. `type` – The type of input. For collectings logs, set this to `filestream`.
+. `streams.id` – A unique identifier for your stream of log data. 
+. `data_stream.dataset` – The name for your dataset data stream. You can name this data stream anything that signifies the source of the data. The default value is `generic`.
+. `paths` – The path to your log files. You can use a pattern like `/var/log/foo.log*`.
```

Comment:
@mdbirnstiehl After the changes were made the config, I think the user has to restart elastic-agent or will it pick up all the changes directly?

Reply:
In the testing I've done, it picks it up without restarting.

Reply:
My assumption is that it picks up changes around the path but not around the output. But I also have seen during my testing that sometimes I felt like it did not pick up changes / not full consistency but could not 100% reproduce it. @cmacknz Does this ring a bell?